### ISSUE TITLE
Step 8.2.3 on Setting Up A Lambda Function Using Amazon Web Services /src path is wrong

### ISSUE THREAD

**Issue by yayimadinosaur** [alexa/alexa-cookbook#37]
current:
\handling-responses\dialog-directive-delegate\src 

correct:
/handling-responses/dialog-directive-delegate/sample-nodejs-plan-my-trip/src

missed one folder

**Comment by SleepyDeveloper** [comment]
Good catch. It's been fixed. I'll be merging this later this afternoon.
